diff --git a/android/src/main/java/expo/modules/audio/AudioModule.kt b/android/src/main/java/expo/modules/audio/AudioModule.kt
index 992239ba7631ce0c97c0b41c5fbf61cc842386fe..ebc0933beaf40414887ff634029ca8eb0d0558c9 100644
--- a/android/src/main/java/expo/modules/audio/AudioModule.kt
+++ b/android/src/main/java/expo/modules/audio/AudioModule.kt
@@ -28,6 +28,9 @@ import androidx.media3.exoplayer.hls.HlsMediaSource
 import androidx.media3.exoplayer.smoothstreaming.SsMediaSource
 import androidx.media3.exoplayer.source.MediaSource
 import androidx.media3.exoplayer.source.ProgressiveMediaSource
+import androidx.media3.extractor.DefaultExtractorsFactory
+import androidx.media3.extractor.mp3.Mp3Extractor
+import androidx.media3.extractor.ts.AdtsExtractor
 import expo.modules.interfaces.permissions.Permissions
 import expo.modules.kotlin.Promise
 import expo.modules.kotlin.exception.Exceptions
@@ -931,7 +934,12 @@ class AudioModule : Module() {
       CONTENT_TYPE_SS -> SsMediaSource.Factory(factory)
       CONTENT_TYPE_DASH -> DashMediaSource.Factory(factory)
       CONTENT_TYPE_HLS -> HlsMediaSource.Factory(factory)
-      CONTENT_TYPE_OTHER -> ProgressiveMediaSource.Factory(factory)
+      CONTENT_TYPE_OTHER -> ProgressiveMediaSource.Factory(
+        factory,
+        DefaultExtractorsFactory()
+          .setAdtsExtractorFlags(AdtsExtractor.FLAG_ENABLE_CONSTANT_BITRATE_SEEKING_ALWAYS)
+          .setMp3ExtractorFlags(Mp3Extractor.FLAG_ENABLE_CONSTANT_BITRATE_SEEKING_ALWAYS)
+      )
       else -> throw IllegalStateException("Unsupported type: $type")
     }.createMediaSource(mediaItem)
   }
diff --git a/expo-module.config.json b/expo-module.config.json
index 9ee5fdcd7a6e698c8cdcf42ee137239220aec190..9a131dd948d2eb5adf3cf9b759829a0e8fed15a1 100644
--- a/expo-module.config.json
+++ b/expo-module.config.json
@@ -4,12 +4,6 @@
     "modules": ["AudioModule"]
   },
   "android": {
-    "modules": ["expo.modules.audio.AudioModule"],
-    "publication": {
-      "groupId": "expo.modules.audio",
-      "artifactId": "expo.modules.audio",
-      "version": "56.0.11",
-      "repository": "local-maven-repo"
-    }
+    "modules": ["expo.modules.audio.AudioModule"]
   }
 }
