diff --git a/src/components/ReanimatedSwipeable/ReanimatedSwipeable.tsx b/src/components/ReanimatedSwipeable/ReanimatedSwipeable.tsx
index 3c4d402aec92fb0d6758428c8c7ed5262a9b3852..a4e1c56e5204c2fd9f413196b249332d37bebc53 100644
--- a/src/components/ReanimatedSwipeable/ReanimatedSwipeable.tsx
+++ b/src/components/ReanimatedSwipeable/ReanimatedSwipeable.tsx
@@ -572,13 +572,13 @@ const Swipeable = (props: SwipeableProps) => {
         {...remainingProps}
         onLayout={onRowLayout}
         style={[styles.container, containerStyle]}>
-        {leftElement()}
-        {rightElement()}
         <GestureDetector gesture={tapGesture} touchAction="pan-y">
           <Animated.View style={[animatedStyle, childrenContainerStyle]}>
             {children}
           </Animated.View>
         </GestureDetector>
+        {leftElement()}
+        {rightElement()}
       </Animated.View>
     </GestureDetector>
   );
